llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -820,6 +820,7 @@ struct AllocMatchInfo {
   uint64_t TotalSize = 0;
   AllocationType AllocType = AllocationType::None;
   bool Matched = false;
+  size_t NumFramesMatched = 0;
```

Comment:
Maybe add this after Total size? Adding it in the end increases the padding size it will be aligned to 8b.